lib/Doctrine/ORM/Configuration.php

```diff
@@ -748,6 +749,60 @@ public function setCustomDatetimeFunctions(array $functions)
         }
     }
 
+    /**
+     * Sets a map of php typed fields to DBAL types for auto-completion.
+     *
+     * Keys must be FQCNs of the PHP types
+     * Values are either FQCNs of DBAL Type or their name
+     *
+     * Any previously added mappings are discarded.
```

Comment:
That does not seem to be what the method does.

Reply:
Hmm, what do you mean? The part that

> Any previously added mappings are discarded.

is indeed confusing, I believe it means to say that previous mapping for the same PHP type is overriden. I copy pasted this from other places, but it seems that this comment is wrong on every single place in the configuration. check
`setCustomStringFunctions`, `setCustomNumericFunctions` and `setCustomDatetimeFunctions`

I would say this is also wrong there, the previous mappings are never discarded, it always add the new ones on top of the existing stack and in case key already exists, it overwrites it with new value.

Should I fix all these comments?

Reply:
> Should I fix all these comments?

If you have the time to do that, that would be great yes! Not a blocker for this PR though. In any case, I don't think consistency should be used to reproduce past errors :wink:

Reply:
I will do this in a separate PR to maintain issue-per-PR isolation